lib/guests.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const Promise = require('bluebird');
+const debug = require('debug')('bot:guests');
+const logError = require('debug')('bot:error');
+
+function guests(bot, message) {
+  debug('begin');
+  let user = message.user;
+  let getData = Promise.promisify(bot.botkit.storage.users.get);
+
+  return getData(user)
+    .then((data) => {
+      debug('pana', data);
+      debug('panas', data.guests);
+
+      if (data.guests.length === 0) {
+        return bot.reply(message, 'Oiga, invitá un parcero primero!');
```

Comment:
We should keep the same level of formality, I think we've been very informal on all commands so the tone should be the same. 

Basically, replace `Oiga`, `oiste` or `oistes` 😂